pkgs/tools/networking/goodhosts/default.nix

```
@@ -0,0 +1,32 @@
+{ lib, buildGoModule, fetchFromGitHub, testers }:
+
+buildGoModule rec {
+  pname = "goodhosts";
+  version = "v1.1.0";
+  name = "goodhosts-${version}";
+
+  src = fetchFromGitHub {
+    owner = "goodhosts";
+    repo = "cli";
+    rev = version;
+    sha256 = "sha256-BsStTOuMercplx+g+egjiBjCKf5UFCDLGqGVo6i5yKQ=";
```

You can use `hash` instead of `sha256`, that is the more idiomatic approach.
```suggestion
    hash = "sha256-BsStTOuMercplx+g+egjiBjCKf5UFCDLGqGVo6i5yKQ=";
```